Support multiple sets of OR'd filters (#634)
WICG/attribution-reporting-api: EVENT.md

  }
}
```

> If keys in the `filters` dictionnary match keys in the `filter_data` dictionnary

Reviewer: ```suggestion
If keys in the `filters` dictionary match keys in the `filter_data` dictionary
```

Reviewer 2: Thanks

and the intersection of their values is empty, the trigger is ignored.

> eg. Given a "conversion_subdomain" key present in both `filter_data` and

Reviewer: ```suggestion
Example: Given a "conversion_subdomain" key present in both `filter_data` and
```

> `filters` dictionnaries. If the values of the `filters`'s "conversion_subdomain"

Reviewer: ```suggestion
`filters` dictionaries. If the values of the `filters`'s "conversion_subdomain"
```

key do not include "electronics.megastore" or "electronics2.megastore", the
trigger gets ignored.

Note: A key which is present in one dictionary and not the other will not be

> included in the matching logic (ie. the trigger will be considered).

Author: Change unrelated to OR'd filters. When first reading on filters, I had to test it to understand how they work.  I'm hoping that this wording makes it a bit clearer. Let me know what you think and/or ifwe'd prefer to have this change in a different PR.

Reviewer: Let's make this change in a separate PR.

Author: https://github.com/WICG/attribution-reporting-api/pull/636

> included in the matching logic (ie. the trigger will be considered).

Reviewer: ```suggestion
included in the matching logic (i.e. the trigger will be considered).
```

}
```

> `filter_data` must be a filter dictionnary. `filters` can be a filter

Reviewer: ```suggestion
`filter_data` must be a filter dictionary. `filters` can be a filter
```

> `filter_data` must be a filter dictionary. `filters` can be a filter
> dictionary or a list of filter dictionaries. When a list is received, only one

Reviewer: ```suggestion
`filter_data` must be a dictionary. `filters` can be a
dictionary or a list of filter dictionaries. When a list is received, only one
```

> dictionnary or a list of filter dictionaries. When a list is received, only one

Reviewer: ```suggestion
dictionary or a list of filter dictionaries. When a list is received, only one
```

> dictionnary has to match for the trigger to be considered.

Reviewer: ```suggestion
dictionary has to match for the trigger to be considered.
```

```jsonc
{

> 
>   "event_trigger_data": [
>     {
>       "trigger_data": "2",
>        "filters": [
>            {"product": ["1234"], "conversion_subdomain": ["electronics.megastore"]},  // OR
>            {"product": ["4321"], "conversion_subdomain": ["electronics4.megastore"]}
>         ]
>     },
> ]}

Reviewer: ```suggestion
  "event_trigger_data": [
    {
      "trigger_data": "2",
      "filters": [
        {"product": ["1234"], "conversion_subdomain": ["electronics.megastore"]},  // OR
        {"product": ["4321"], "conversion_subdomain": ["electronics4.megastore"]}
      ]
    }
  ]
}
```
